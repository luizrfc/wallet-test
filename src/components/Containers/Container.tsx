import { View } from "react-native"
import { styleTheme } from "../../styles/themeStyle"

interface ContainerProps {
    children: React.ReactNode
}

const Container = ({ children }: ContainerProps): JSX.Element => {
    return <View style={styleTheme(16).main}>
        {children}
    </View>
}

export default Container;