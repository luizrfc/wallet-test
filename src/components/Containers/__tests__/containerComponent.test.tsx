import React, { Fragment } from "react";
import "react-native";
import renderer from "react-test-renderer";
import Container from "../container";

test("Render a container component correctly", () => {
  const button = renderer
    .create(
      <Container>
        <Fragment></Fragment>
      </Container>
    )
    .toJSON();
  expect(button).toMatchSnapshot();
});
