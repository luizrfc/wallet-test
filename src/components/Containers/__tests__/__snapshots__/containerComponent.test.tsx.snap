// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render a container component correctly 1`] = `
<View
  style={
    {
      "alignItems": "center",
      "backgroundColor": "#142995",
      "flex": 1,
      "justifyContent": "center",
      "padding": 16,
    }
  }
/>
`;
